cypress/integration/page.spec.js

```diff
@@ -1086,7 +1086,7 @@ describe('', () => {
         .should('have.text', 'Some comment body');
     });
 
-    it('should show submit button spinner while waiting for server response', () => {
+    it.skip('should show submit button spinner while waiting for server response', () => {
```

Comment:
The test 'should show submit button spinner while waiting for server response' is currently skipped. Ensure this test is completed and passing before including it in the test suite.